libs/message-handler/tsconfig.node.json

```
@@ -18,5 +18,5 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src"]
+  "include": ["src", "src/pickup.test.ts"]
 }
```

including the src directory is enough here 
```suggestion
  "include": ["src"]
}
```